Controllers/ServerEventsController.cs

```diff
@@ -28,13 +28,12 @@ public ServerEventsController(ILogger<ServerEventsController> logger, IEnumerabl
 			_eventFinalizers = eventFinalizers;
 			_logger.LogInformation("DI provided {evtFinalizerCount} event finalizers.", _eventFinalizers.Count().ToString());
 		}
-
 		/// <summary>
 		/// Provide a GET endpoint for connecting to the SSE channel
 		/// </summary>
 		/// <returns>Good question</returns>
 		[HttpGet()]
-		public async Task SSE() {
+		public async Task SSE(CancellationToken cancellationToken = default) {
```

Comment:
Is it necessary to make this an optional argument? In fact, should it be an optional agrument?

Reply:
You're right, omitting the `= default` doesn't seem to change anything, thanks.